sed/core/logging.py

```diff
@@ -51,24 +82,26 @@ def setup_logging(
     # Add console handler to logger
     logger.addHandler(console_handler)
 
-    # Determine log file path
-    os.makedirs(user_log_path, exist_ok=True)
-    log_file = os.path.join(user_log_path, f"sed_{datetime.now().strftime('%Y-%m-%d')}.log")
+    # Capture warnings with the logging system
+    logging.captureWarnings(True)
 
-    # Create file handler and set level to debug
-    file_handler = logging.FileHandler(log_file)
-    file_handler.setLevel(FILE_VERBOSITY)
+    return logger
 
-    # Create formatter for file
-    file_formatter = logging.Formatter(
-        "%(asctime)s - %(name)s - %(levelname)s - %(message)s in %(filename)s:%(lineno)d",
-    )
-    file_handler.setFormatter(file_formatter)
 
-    # Add file handler to logger
-    logger.addHandler(file_handler)
+def call_logger(logger: logging.Logger):
```

Comment:
I like the idea of logging calls. As debug it also seems good. But we can additionally also add this info to the metadata (only the function call and parameters, excluding other details)

Reply:
Yes, this can be something for the future, to build up a call stack to reproduce a pipeline, similar to what @steinnymir had in mind.

Reply:
In favor of not enlarging this PR, I create an issue for this.